addons/website_forum/views/forum.xml

```diff
@@ -32,6 +69,37 @@
             <field name="arch" type="xml">
                 <form string="Forum">
                     <sheet>
+                        <div class="oe_button_box" name="button_box">
+                            <button name="%(action_forum_posts)d" type="action" class="oe_stat_button" icon="fa-comments">
+                                <div class="o_form_field o_stat_info">
+                                    <span class="o_stat_value">
+                                        <field name="total_posts" />
+                                    </span>
+                                    <span class="o_stat_text">Posts</span>
+                                </div>
+                            </button>
+                            <button name="%(action_forum_answers)d" type="action" class="oe_stat_button" icon="fa-reply">
+                                <div class="o_form_field o_stat_info">
+                                    <span class="o_stat_value">
+                                        <field name="total_answers" />
+                                    </span>
+                                    <span class="o_stat_text">Answers</span>
+                                </div>
+                            </button>
+                            <button name="%(action_forum_favorites)d" class="oe_stat_button" icon="fa-star" type="action">
+                                <div class="o_form_field o_stat_info">
+                                    <span class="o_stat_value">
+                                        <field name="total_favorites" />
+                                    </span>
+                                    <span class="o_stat_text">Favorites</span>
+                                </div>
+                            </button>
```

Comment:
After reconsidering, I don't see the added value for those 2 stats buttons. So I cancel: https://tinyurl.com/ybbxkvum